OpenDreamRuntime/Procs/DMProc.cs

```diff
@@ -543,6 +543,7 @@ public override void Dispose() {
             _stack = null;
 
             _dreamValuePool.Return(_localVariables);
+            Array.Clear(_localVariables);
```

Comment:
`ArrayPool.Return()` has a `clearArray` parameter.
```suggestion
            _dreamValuePool.Return(_localVariables, true);
```